protobuf/transform.go

```diff
@@ -324,7 +324,11 @@ func (t *Transformer) transformType(pkg *Package, typ scanner.Type, msg *Message
 		if field.Options == nil {
 			field.Options = make(Options)
 		}
-		field.Options["(gogoproto.casttype)"] = NewStringValue(castType(pkg, n.Type))
+
+		// Repeated types cannot use casttype :(
```

Comment:
Not an issue, just the problem between repeated being part of the field in protobuf and part of the type in Go.